requirements/index.html

```diff
@@ -0,0 +1,289 @@
+			<li>Improve the ability to support automated testing where appropriate and provide a procedure for repeatable tests when manual testing is appropriate.</li>
+			<li>Provide requirements to address functional needs, while avoiding or minimizing negative impacts on other functional needs. </li>
+			<li>Include accompanying documents that provide clear, concise, and findable technical information quickly in one place.</li>
+			<li>Include, where possible and appropriate, links to instruction videos, illustrations, and how-to guides. Creation of new videos and illustrations are outside the scope of the working group at this time.</li>
+			<li>Normative requirements should be unambiguous so there is only one possible meaning for each requirement.</li>
+          </ol>
+          <p>The creation process for the guidelines should:</p>
+          <ol start="7">
+            <li>Actively recruit a diverse range of people with disabilities in recognition of the importance of their contributions to accessibility standards and solutions. Review and monitor whether people are included. Continually evaluate inclusive features of available tooling and procedures. </li>
+            <li>Facilitate global participation and feedback.</li>
+            <li>Be data-informed and evidence-based where possible.  We recognize that research and evidence are influenced by the number of people with a particular disability, by the size of the body of research, and by the difficulty in capturing data regarding some disabilities or combination of disabilities.  The intent is to make informed decisions wherever possible to ensure that the needs of all people with disabilities are prioritized, including needs that differ from the majority. In situations where there is no evidence or research, valid data-gathering methods should be used to obtain and evaluate information from advocacy groups, people with lived experience and other subject matter experts.</li>
+			<li>Be written so the Guideline content is usable in adaptable and customizable ways. For example, WCAG 3.0 content is available to be extracted by users to adapt to their needs. </li>			
+          </ol>
+</section>
+		<section>
+		  <h2 id="requirements">Requirements</h2>
+          <p>Previous W3C Accessibility Guidelines described how to make web    pages accessible to people with disabilities. These guidelines provided a     flexible framework that has kept the guidelines relevant for   10 years. Changing technology and changing needs of people with   disabilities   has shown areas where they could be improved. The   requirements are   drawn from the research performed by WCAG 3.0 to   improve the guidelines,   and the suggestions from the Silver Design  Sprint. </p>
```

Comment:
There is an extra space between for and 10 years. There are also extra spaces before The and requirements, are and drawn and guidelines, and.